docs/proposals/updates-first-pass.md

```diff
@@ -0,0 +1,205 @@
+# Initial APB Update support
+
+[OBS Update Spec](https://github.com/openservicebrokerapi/servicebroker/blob/v2.12/spec.md#updating-a-service-instance)
+
+## Introduction
+
+Updates allow APB authors to mutate existing service instances.
+
+> By implementing this endpoint, service broker authors can enable users
+to modify two attributes of an existing service instance: the service plan
+and parameters. By changing the service plan, users can upgrade or downgrade
+their service instance to other plans. By modifying properties, users can change
+configuration options that are specific to a service or plan
+
+Therefore, the cases we're focused on are:
+
+1) Changing the argument used for a given parameter. Ex: change a "name"
+
+2) Moving from one plan to another.  Ex: Move from non-persistent plan to
+a persisted plan. Another example of this could be moving from a multi-tenant
+environment to a dedicated environment for a service's database.
+
+NOTE: The Service Catalog currently does **not** implement update, so triggering
+these from the UI is not possible. Updates should be considered very "alpha".
+To demonstrate , the plan is to enhance `asbcli` to support update.
```

Comment:
Update is a [distinct operation/request](https://github.com/openservicebrokerapi/servicebroker/blob/v2.12/spec.md#updating-a-service-instance) the platform (service-catalog in this case) is expected to implement. It is not a re-provision. @jmrodri any idea on a timeline? I was not present for the meeting where this came up.